init.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+if [ "${BRANCH_ROLE}" != "source" ] && [ "${BRANCH_ROLE}" != "staging" ] && [ "${BRANCH_ROLE}" != "live" ] && [ "${BRANCH_ROLE}" != "custom" ]; then
+    echo "usage: BRANCH_ROLE=<source|staging|live|custom> ./init.sh"
+    exit 1
+fi
+
+# cd into the Git repository that you are converting into a charts repository
```

Comment:
this comment looks out of place

Reply:
ack, I left that behind on accident. Removed.